pytheus/metrics.py

```diff
@@ -202,7 +217,7 @@ def _raise_if_cannot_observe(self) -> None:
         if not self._can_observe:
             raise UnobservableMetricException
 
-    def labels(self, pytheus_labels_: Optional[Labels] = None, **kwargs: str) -> "_Metric":
+    def labels(self, pytheus_labels_: Optional[Labels] = None, **kwargs: str) -> "Self":
```

Comment:
If the suggestion from this review gets applied, I would remove the quotes and use directly `Self`